src/Microsoft.PowerShell.Commands.Management/commands/management/SetClipboardCommand.cs

```diff
@@ -52,7 +54,25 @@ public class SetClipboardCommand : PSCmdlet
         [Parameter(ParameterSetName = PathParameterSet, Mandatory = true, ValueFromPipelineByPropertyName = true)]
         [ValidateNotNullOrEmpty]
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
-        public string[] Path { get; set; }
+        public string[] Path
+        {
+            get
+            {
+                return _path;
+            }
+
+            set
+            {
+#if UNIX
+                ThrowTerminatingError(new ErrorRecord(new InvalidOperationException(ClipboardResources.PathUnsupported),
+                    "FailedToSetClipboard", ErrorCategory.InvalidOperation, "Clipboard"));
```

Comment:
If Path is a string why don't we support this?

Reply:
On Windows, file paths get stored into a separate part of the Windows clipboard so pasting to an app that understands it treats it as an object.  I don't think we want different behavior on non-Windows to just treat it as a string.